public/js/forum.js

```diff
@@ -86,3 +86,56 @@ function dislikeComment(event, commentId) {
         'dataType': 'json'
     })
 }
+
+// This works...
+function displayEdit() {
+    $("#editForum").toggle();
+    $("#viewForum").toggle();
+}
+
+$(document).ready(function() {
+    $("#updateForum").click(function (event) {
+        displayEdit();
+        let forumId = $("#forumId").val();
+        let title = $("#newForumTitle").val();
+        let content = $("#newForumContent").val();
+        let labels = null;
+        $.ajax({
+            'url': '/forums/' + forumId,
+            'type': 'PUT',
+            'success': function (res) {
+                console.log("Successfully updated forum " + forumId);
+                $("#forumTitle").text(title);
+                $("#forumContent").text(content);
+            },
+            'error': function (err) {
+                if (err) {
+                    console.log(err);
+                }
+            },
+            'data': {
+                "title": title,
+                "content": content,
+                "labels": labels
```

Comment:
Why pass labels if it will always be null?

Reply:
Now can update labels.